### PR TITLE
EPMRPP-79834 || Popover reuse settings

### PR DESCRIPTION
**Pull request by tr1ble**

### CODE REVIEW

File: app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.jsx [hunk 1]
```diff
@@ -74,24 +74,32 @@ const messages = defineMessages({
   },
 });
 
-const Tooltip = () => {
-  const { formatMessage } = useIntl();
-  return <div>{formatMessage(messages.tooltip)}</div>;
+const Popover = ({ formatMessage }) => (
+  <div className={cx('popover')}>
+    <div className={cx('title')}>{formatMessage(messages.showFieldsHeader)}</div>
```
Comment (Vadim73i) [inline app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.jsx:79]: Can we use withPopover title prop for title?

File: app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.jsx [hunk 2]
```diff
@@ -74,24 +74,32 @@ const messages = defineMessages({
...
-  data: {
-    dynamicWidth: true,
-    align: 'bottom',
-    noArrow: false,
-    dark: true,
+const ShowWithTooltip = withPopover({
+  ContentComponent: Popover,
+  side: 'bottom',
+  arrowPosition: 'middle',
+  popoverWrapperStyle: {
```
Comment (Vadim73i) [inline app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.jsx:95]: use popoverWrapperClassName instead popoverWrapperStyle

**Review by Vadim73i — CHANGES_REQUESTED** [review]

File: app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.jsx (hunk 1 above)
Comment (tr1ble) [reply app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.jsx:79]: I'm not sure that we can pass localizied title to `ShowWithTooltip`

File: app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.scss [hunk 3]
```diff
@@ -75,3 +75,20 @@
     display: none;
   }
 }
+
+.popover {
+  font-family: $FONT-ROBOTO-REGULAR;
+  font-style: normal;
+  font-weight: 400;
```
Comment (AmsterGet) [inline app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.scss:82]: These properties (`font-style`, `font-weight`) are normal by default for all fonts, please remove them.

File: app/src/pages/inside/projectSettingsPageContainer/content/defectTypes/defectTypeRow/defectTypeRow.scss [hunk 4]
```diff
@@ -132,6 +132,7 @@ $DEFECT_NAME_PADDING: 16px;
 
 .defect-locator-tooltip {
```
Comment (AmsterGet) [inline app/src/pages/inside/projectSettingsPageContainer/content/defectTypes/defectTypeRow/defectTypeRow.scss:133]: May be update the class name?
Seems like it is not a tooltip any more.

File: app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.jsx [hunk 5]
```diff
@@ -74,24 +74,28 @@ const messages = defineMessages({
...
 
-const ShowWithTooltip = withTooltip({
-  TooltipComponent: Tooltip,
-  data: {
-    dynamicWidth: true,
-    align: 'bottom',
-    noArrow: false,
-    dark: true,
-  },
+const ShowWithTooltip = withPopover({
```
Comment (AmsterGet) [inline app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.jsx:91]: Same here about component name.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**